styles/globals.scss

```diff
@@ -484,3 +484,7 @@
 .fs-1 {
   font-size: 0.8rem;
 }
+
+.on-scrollbar::-webkit-scrollbar-thumb {
+  background: var(--scrollbar-thumb-color) !important;
```

Comment:
do NOT use !important, you may move this class into .main class and it will work without !important